XmlFormatter/src/Windows/Settings.cs

```diff
@@ -89,5 +96,84 @@ private void B_Cancel_Click(object sender, EventArgs e)
         }
+
+        /// <summary>
+        /// This method will export the settings of the application so you can use it on another computer
+        /// </summary>
+        /// <param name="sender">The sender wo called the event</param>
+        /// <param name="e">The arguments provided by the sender</param>
+        private void exportSettingsToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            SaveFileDialog saveFileDialog = new SaveFileDialog();
+            saveFileDialog.Filter = "XML files(*.xml)| *.xml";
+            saveFileDialog.FileName = DateTime.Now.ToString("yyyyMMdd_HHmmss");
+            saveFileDialog.FileName += "_";
+            saveFileDialog.FileName += Application.ProductName;
+            saveFileDialog.FileName += "Settings";
+            saveFileDialog.FileName += "_V";
+            saveFileDialog.FileName += Properties.Settings.Default.ApplicationVersion;
+
+            DialogResult result = saveFileDialog.ShowDialog();
+
+            if (result != DialogResult.OK)
+            {
+                return;
+            }
+
+            WriteSettings();
+            Configuration configuration = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.PerUserRoamingAndLocal);
+            configuration.SaveAs(saveFileDialog.FileName);
+        }
+
+        /// <summary>
+        /// This method will allow you to import the exported settings back into the application
+        /// </summary>
+        /// <param name="sender">The sender wo called the event</param>
+        /// <param name="e">The arguments provided by the sender</param>
+        private void importSettingsToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            OpenFileDialog openFileDialog = new OpenFileDialog();
+            openFileDialog.Filter = "XML files(*.xml)| *.xml";
+            DialogResult result = openFileDialog.ShowDialog();
+            if (result != DialogResult.OK)
+            {
+                return;
+            }
+
+            try
+            {
+                XDocument settingsDocument = XDocument.Load(openFileDialog.FileName);
+                XNode versionNode = settingsDocument.XPathSelectElement("//userSettings//setting[@name='ApplicationVersion']//value");
+                XElement Version = XElement.Parse(versionNode.ToString());
+                string loadedVersion = Version.Value;
+                VersionManager versionManager = new VersionManager();
+                Version importFileVersion = versionManager.ConvertInnerFormatToProperVersion(loadedVersion);
+                if (importFileVersion != versionManager.GetApplicationVersion())
+                {
+                    string message = "Application version is not matching settings version";
+                    message += "\r\n\r\nApplication Version: " + versionManager.GetApplicationVersion().ToString();
+                    message += "\r\nSettings Version: " + importFileVersion.ToString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There's no need to call 'ToString()', the compiler will do it for you.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5002950)